src/main/kotlin/com/craftschemers/hub/managers/HubMinigameManager.kt

```diff
@@ -1,41 +1,36 @@
 package com.craftschemers.hub.managers
 
 import com.craftschemers.hub.HubPlayer
-import com.craftschemers.hub.minigame.GameState
 import com.craftschemers.hub.minigame.GameType
-import com.craftschemers.hub.minigame.Minigame
-import com.craftschemers.hub.minigame.OneInTheQuiver
+import com.craftschemers.hub.minigame.oitq.OneInTheQuiver
+import com.craftschemers.hub.minigame.sendErrorMessage
 
 class HubMinigameManager {
 
-    val minigameServers = mutableMapOf<GameType, List<Minigame>>()
+    val minigames = mapOf(
+        GameType.ONE_IN_THE_QUIVER to OneInTheQuiver
```

Comment:
Trailing comma please
```suggestion
        GameType.ONE_IN_THE_QUIVER to OneInTheQuiver,
```